### DISCUSSION TITLE
maskpxdata only updating first frame

### DISCUSSION THREAD

**Discussion by ScreamingInDigital** [dcm4che/dcm4che#1298] [Q&A] [answered]
Has anyone had an issue with the "maskpxdata" only updating the first frame?

I used "dcm2dcm" to uncompress a single file followed by using "maskpxdata" to remove the banner generated on a Siemens S2000.  It only removed the banner on the first frame.  The documentation does not show anything about specifying all frames or specific frames.

**Comment by gunterze** [comment, marked as answer]
Fixed by #1343
